include/opcua.h

```diff
@@ -122,6 +123,7 @@ class OPCUA
         std::string            	m_clientPrivate;
         std::string            	m_certAuth;
         std::string            	m_caCrl;
+	struct timeval		m_userts;
```

Comment:
The OPCUA class represents a client connection to an OPC UA server. It is not the right place to put a member that is used to construct a single Reading.

Reply:
fixed